lib/services/credentials/credential.service.js

```diff
@@ -143,8 +143,26 @@ s.getCredential = function (id, type, options) {
     });
 };
 
-s.getAllCredentials = function (consumerId) {
-  return credentialDao.getAllCredentials(consumerId);
+s.getCredentials = function (consumerId, options = {}) {
+  return credentialDao
+    .getAllCredentials(consumerId)
+    .then((credentials) => {
+      const include = Array.isArray(options.include) ? options.include : [];
+      include.sort();
+      let filter;
+      switch (include.join(',')) {
+        case 'active,archive':
```

Comment:
ha-ha let's hope nobody will pass arguments in different order 🥇 (I see the sort but no test for different order of arguments)

I assumed that it would output active always and if include is present as "archived" 
it will output everything. because include is kind of add to something you already have.

Now I see the task says to have 3 way, so `filter` may be better word than include. sorry for confusion

Reply:
for the code part, I suggest something explicit: 
```js
const showActive = 'active' in params.filter
const showArchived = 'archived' in params.filter
if (showActive && !showArchived) {
    return credentials.fillter(c=>c.isActive)
}
if (!showActive && showArchived) {
    return credentials.fillter(c=>!c.isActive)
}
return credentials //default

```